runtime-config.md

```diff
@@ -1,3 +1,8 @@
+# Host-specific container configuration
+
+The bundle's top-level directory MUST contain a configuration file called `runtime.json` with [UTF-8][] [JSON][].
+For now the canonical schema is defined in [`runtime_config.go`](runtime_config.go) and [runtime_config_linux.go](runtime_config.go), but this will be moved to a formal JSON schema over time.
```

Comment:
I would have the OS specific reference be more generic than pointing out linux individually.